### PR TITLE
Adding tests by weilinms

### PR DESCRIPTION
**Pull request by ProbstenHias**

### CODE REVIEW

**Comment by ProbstenHias** [comment]
I changed the project structure a little since the command `pytest` gave me an error saying the module `diffusion2s` can not be found. 

- I added `__init__.py` files to all directories
- I changed the import paths in the test files to relative imports

Now I know that this was not necessary and the command `python -m pytest` works without those changes.
Let me know if I should change this back.

File: tests/integration/test_diffusion2d.py [hunk 1]
```diff
@@ -2,18 +2,55 @@
 Tests for functionality checks in class SolveDiffusion2D
 """
 
-from diffusion2d import SolveDiffusion2D
+import numpy as np
+from ...diffusion2d import SolveDiffusion2D
```
Comment (IshaanDesai) [inline tests/integration/test_diffusion2d.py:6]:
```suggestion
from diffusion2d import SolveDiffusion2D
```
Relative import did not work on my machine when I tried to run the tests. Nothing to worry, just fyi.

File: tests/unit/test_diffusion2d_functions.py [hunk 2]
```diff
@@ -2,25 +2,53 @@
 Tests for functions in class SolveDiffusion2D
 """
 
-from diffusion2d import SolveDiffusion2D
-
-
-def test_initialize_domain():
+from ...diffusion2d import SolveDiffusion2D
```
Comment (IshaanDesai) [inline tests/unit/test_diffusion2d_functions.py:5]:
```suggestion
from diffusion2d import SolveDiffusion2D
```
Same as above.

**Review by IshaanDesai — APPROVED** [review]
Adding `__init__.py` files to convert folders into modules is not wrong, but sort of a solution to get things to work. No need to change anything back.

**Pull request closed without merging**